File: sentimental-mario-less/mario.py
# TODO
from cs50 import get_int

while True:
    try:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            break
    except:
        print("Error!")
        exit()

for i in range(1, height + 1):
    print(" " * (height - i) + "#" * i)

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int a, b, c;
    do
    {
        a = get_int("Height: ");
    }
    while (a < 1 || a > 8);

    for (b = 1; b <= a; b++)
    {
        for (c = 1; c <= a - b; c++)
        {
            printf(" ");
        }

        for (c = 1; c <= b; c++)
        {
            printf("#");
        }
        printf("\n");
    }
}